goals/Nov-16.md

```diff
@@ -0,0 +1,11 @@
+## Goals for Nov 17 - Nov 22
```

Comment:
I think you should remove this file from the branch. It's already present in a separate branch.